packages/wdio-sauce-service/src/index.js

```diff
@@ -1,72 +1,25 @@
 import request from 'request'
-import SauceConnectLauncher from 'sauce-connect-launcher'
+import logger from 'wdio-logger'
 
 const jobDataProperties = ['name', 'tags', 'public', 'build', 'custom-data']
 
 const jasmineTopLevelSuite = 'Jasmine__TopLevel__Suite'
 
-export default class SauceService {
-    /**
-     * modify config and launch sauce connect
-     */
-    onPrepare (config, capabilities) {
-        if (!config.sauceConnect) {
-            return
-        }
-
-        this.sauceConnectOpts = Object.assign({
-            username: config.user,
-            accessKey: config.key
-        }, config.sauceConnectOpts)
-
-        config.protocol = 'http'
-        config.host = 'localhost'
-        config.port = this.sauceConnectOpts.port || 4445
-
-        const sauceConnectTunnelIdentifier = this.sauceConnectOpts.tunnelIdentifier
-
-        if (sauceConnectTunnelIdentifier) {
-            if (Array.isArray(capabilities)) {
-                capabilities.forEach(capability => {
-                    capability.tunnelIdentifier = capability.tunnelIdentifier || sauceConnectTunnelIdentifier
-                })
-            } else {
-                Object.keys(capabilities).forEach(browser => {
-                    capabilities[browser].desiredCapabilities.tunnelIdentifier = capabilities[browser].desiredCapabilities.tunnelIdentifier || sauceConnectTunnelIdentifier
-                })
-            }
-        }
-
-        return new Promise((resolve, reject) => SauceConnectLauncher(this.sauceConnectOpts, (err, sauceConnectProcess) => {
-            if (err) {
-                return reject(err)
-            }
-
-            this.sauceConnectProcess = sauceConnectProcess
-            return resolve()
-        }))
-    }
-
-    /**
-     * shut down sauce connect
-     */
-    onComplete () {
-        if (!this.sauceConnectProcess) {
-            return
-        }
+const log = logger('wdio-sauce-service')
 
-        return new Promise(resolve => this.sauceConnectProcess.close(resolve))
+export default class SauceService {
+    constructor (config) {
+        this.config = config
+        this.isMultiremote = !Array.isArray(config.capabilities)
```

Comment:
The capabilities for multiremote can be defined as an object as well right?
```
capabilities: {
  browserA: {},
  browserB: {}
}
```

Reply:
yes, which is why `isMultiremote` is true if `capabilities` is not an array